tests/features/bootstrap/FeatureContext.php

```diff
@@ -425,7 +426,10 @@ public function assertFieldOptionSelected($option, $select) {
    */
   public function assertFieldRadioSelected($radio, $field) {
     // Find the grouping fieldset that contains the radios field.
-    $fieldset = $this->getSession()->getPage()->find('named', ['fieldset', $field]);
+    $fieldset = $this->getSession()->getPage()->find('named', [
```

Comment:
Please revert this change, it makes it less readable.

Reply:
For this file, I did not notice that it was not scanned by phpcs. I will revert all non-functional changes. I guess I clicked an auto formatter and these were changed..

Reply:
I've noticed PHPStorm doing this even without pressing any buttons.

A good tip is to use `git add -p` when adding files to the staging area, this allows you do to a mini-review of the changes that are being added. It's really helpful for catching these auto-formatted lines.